README.md

```diff
@@ -459,6 +459,113 @@ for (const transactionSummary of transactionSummaries) {
 }
 ```
 
+## Deploy module
+The following example demonstrates how to deploy a smart contract module.
+```js
+/**
+ *
+ * @param filePath for the wasm file moudule
+ * @returns Buffer of the wasm file
+ */
+function getByteArray(filePath: string): Buffer {
+    const data = fs.readFileSync(filePath);
+    return Buffer.from(data);
+}
+//To get the buffer of the wasm file from the previous method
+const wasmFileBuffer = getByteArray(wasmFilePath) as Buffer;
+
+const deployModule: DeployModulePayload = {
+    tag: ModuleTransactionType.DeployModule,
+    content: wasmFileBuffer,
+    length: wasmFileBuffer.length,
+    version: 0,
+} as DeployModulePayload;
+
+let deployModuleTransaction: AccountTransaction;
+ 
+const header: AccountTransactionHeader = {
+    expiry: new TransactionExpiry(new Date(Date.now() + 3600000)),
+    nonce: nextAccountNonce.nonce,
+    sender: new AccountAddress(senderAccountAddress),
+};
+
+const deployModuleTransaction: AccountTransaction = {
+    header: header,
+    payload: deployModule as AccountTransactionPayload,
+    type: AccountTransactionType.DeployModule,
+};
+```
+
+Finally, to actually deploy the module to the chain, send the constructed `deployModuleTransaction` to the chain using `sendAccountTransaction`. (See [Send Account Transaction](#Send-Account-Transaction) for how to do this)
+
+## Init Contract (parameterless smart contract)
+The following example demonstrates how to initialize a smart contract from a module, which has already been deployed. 
+The name of the contract should be specified, and should include "init_" prefix (So if the contract should be named 'INDBank', then the initName should be "init_INDBank".
+In this example, the contract does not take any parameters, so we can leave params as an empty list.  
+```js
+const initName = 'init_INDBank'; 
+const params = [];
```

Comment:
@orhoj We will try keep this as it is for now since we have introduced params as different interface where we introduced may things at code so it will be helpful in the next Pull Request but not in this PR